test/go.mod

```
@@ -130,16 +130,16 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.28.5 // indirect
-	k8s.io/apiserver v0.28.5 // indirect
-	k8s.io/client-go v0.28.5 // indirect
+	k8s.io/apiextensions-apiserver v0.29.2 // indirect
```

```suggestion
	k8s.io/apiextensions-apiserver v0.29.3 // indirect
```
All these should be the same too.